diff --git a/node_modules/@babel/eslint-parser/lib/index.cjs b/node_modules/@babel/eslint-parser/lib/index.cjs
index bc8ac80..c1afb0b 100644
--- a/node_modules/@babel/eslint-parser/lib/index.cjs
+++ b/node_modules/@babel/eslint-parser/lib/index.cjs
@@ -15,18 +15,7 @@ const {
 
 const convert = require("./convert/index.cjs");
 
-const babelParser = require((((v, w) => (v = v.split("."), w = w.split("."), +v[0] > +w[0] || v[0] == w[0] && +v[1] >= +w[1]))(process.versions.node, "8.9") ? require.resolve : (r, {
-  paths: [b]
-}, M = require("module")) => {
-  let f = M._findPath(r, M._nodeModulePaths(b).concat(b));
-
-  if (f) return f;
-  f = new Error(`Cannot resolve module '${r}'`);
-  f.code = "MODULE_NOT_FOUND";
-  throw f;
-})("@babel/parser", {
-  paths: [require.resolve("@babel/core/package.json")]
-}));
+const babelParser = require("@babel/parser");
 
 let isRunningMinSupportedCoreVersion = null;
 
